controller/tap/server.go

```diff
@@ -30,23 +29,24 @@ const requireIDHeader = "l5d-require-id"
 const podIPIndex = "ip"
 const defaultMaxRps = 100.0
 
-type (
-	server struct {
-		tapPort             uint
-		k8sAPI              *k8s.API
-		controllerNamespace string
-	}
-)
+// GRPCTapServer describes the gRPC server implementing tap.Tap_TapByResourceServer
```

Comment:
I think this comment should say `pb.TapServer` instead of `tap.Tap_TapByResourceServer`